samples/gibbs_ensemble/run_sim.py

```diff
@@ -0,0 +1,517 @@
+#
+# Copyright (C) 2013-2019 The ESPResSo project
+#
+# This file is part of ESPResSo.
+#
+# ESPResSo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ESPResSo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Simulate a Gibbs-ensemble of a Lennard-Jones fluid at a fixed temperature.
+This script does the Monte-Carlo part of the Gibbs-ensemble, however for the
+energy calculation of the systems, two instances of the Client class in
+:file:`gibbs_ensemble.py` initialized (and possibly updated) in
+:file:`gibbs_client_system.py` are started which each run a different instance
+of ESPResSo.
+
+The Gibbs-ensemble implemented in these scripts closely refers to chapter 8 of
+Daan Fenkel and Berend Smit 'Understanding Molecular Simulation, Second edition'.
+All equation references noted in this script can be found there.
+
+Due to the cutoff and shifting of the LJ-potential the simulated points in the
+phase diagram differ from the long range uncut LJ-potential. The phase diagram
+of the used potential can be found in 'B. Smit, J. Chem. Phys. 96 (11), 1992,
+Phase diagrams of Lennard-Jones fluids'.
+"""
+
+from multiprocessing import Pipe
+import numpy as np
+from tqdm import tqdm
+import gzip
+import pickle
+import argparse
+import logging
+from enum import Enum, unique
+
+from gibbs_ensemble import MessageID
+from client_system import Gibbs_Client
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('T', type=float, help="Temperature of the system")
+parser.add_argument(
+    '-N',
+    type=int,
+    default=256,
+    help="Number of particles, default 256")
+parser.add_argument(
+    '-density',
+    type=float,
+    default=0.3,
+    help="Init density, default 0.3")
+parser.add_argument(
+    '-steps',
+    '-s',
```

Comment:
Here and elsewhere, it is quite common to have two dashes for the long name and a single dash for the abbreviation, i.e. `--steps / -s`.

Reply:
...but then you also need to adapt `samples/gibbs_ensemble/run_for_several_kT.sh` at line 19 :-)